playbooks/provisioning/openstack/README.md

```diff
@@ -118,6 +118,14 @@ The `required_packages` variable also provides a list of the additional
 prerequisite packages to be installed before to deploy an OpenShift cluster.
 Those are ignored though, if the `manage_packages: False`.
 
+The `inventory` controls either a static inventory will be created after the
+cluster nodes provisioned on OpenStack cloud. Note, the fully dynamic inventory
+is yet to be supported, so the static inventory will be created anyway.
+
+The `inventory_dir` points the directory to host the generated static inventory.
```

Comment:
You can remove this paragraph. Ansible sets `inventory_dir` automaticaly:

http://docs.ansible.com/ansible/playbooks_variables.html

It's not something the user needs to set up. It will just always point to the inventory directory.